.github/workflows/Tests.yaml

```diff
@@ -394,9 +394,6 @@ jobs:
       env:
         # We run unit tests with the following compiler flags:
         # - `-Werror`: Treat warnings as error.
-        # - `-march=x86-64`: Make sure we are building on a consistent
-        #   architecture so caching works. This is necessary because GitHub
-        #   may run the job on different hardware.
```

Comment:
Are you sure this is incorrect? We do set `OVERRIDE_ARCH` below

Reply:
I guess technically the comment is correct. But, we don't set this as an `env` variable anymore. It's hard-coded into most of the cmake commands (except for our architecture checks), so it doesn't seem like this is the correct place for the comment. Should I move it somewhere else?

Reply:
I see. I think it would be good to keep the comment about caching and GitHub. I'd say just leave the comment here unless you can find a better place for it.

Reply:
Ok I'll remove that commit and just leave the comment in.